### PR TITLE
Added Employee Database and some styling

### PR DESCRIPTION
**Pull request by danielsteenberg**
Created employee database and features with styling

### CODE REVIEW

File: app/assets/stylesheets/form.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+.form
```
Comment (atainton) [inline app/assets/stylesheets/form.scss]: ?

File: app/assets/stylesheets/inputform.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.form {
```
Comment (atainton) [inline app/assets/stylesheets/inputform.scss:1]: indentation is way off here

File: app/assets/stylesheets/inputform.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	border: solid #000000;
+	Width: 50%;
```
Comment (atainton) [inline app/assets/stylesheets/inputform.scss]: lower case

File: app/views/employees/edit.html.slim [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.container
```
Comment (atainton) [inline app/views/employees/edit.html.slim:1]: indentation

File: app/views/employees/edit.html.slim [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+	h1 Editing employee
+
+	.float-left
+		== render 'form'
+
+	= button_to 'Show', @employee, :method => :get
```
Comment (atainton) [inline app/views/employees/edit.html.slim]: please use link_to

File: app/views/employees/index.html.slim [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      thead
+        tr
+          th Fist Name
+          th Last Name
+          th
+
+      tbody
+        - @employees.each do |employee|
+          tr
+            td = employee.fname
```
Comment (atainton) [inline app/views/employees/index.html.slim]: what is fname, this should be more descriptive

File: app/views/employees/index.html.slim [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+            td = employee.lname
+            td 
+              = button_to 'view', employee, :method => :get
```
Comment (atainton) [inline app/views/employees/index.html.slim]: please use link_to

File: app/views/employees/new.html.slim [hunk 8]
```diff
@@ -0,0 +1,7 @@
+.container
+	h1 New Employee
+
+	.form
+	  == render 'form'
```
Comment (atainton) [inline app/views/employees/new.html.slim]: double equal?

File: app/views/shared/_nav.html.slim [hunk 9]
```diff
@@ -23,7 +23,7 @@ nav class="navbar navbar-inverse navbar-fixed-top" role="navigation"
           a href="contact.html"Contact
 
         li
-          a href="/employee/employee"Employees
+          a href="/employees"Employees
```
Comment (atainton) [inline app/views/shared/_nav.html.slim]: employees_path

File: app/views/shared/_nav.html.slim [hunk 10]
```diff
@@ -23,7 +23,7 @@ nav class="navbar navbar-inverse navbar-fixed-top" role="navigation"
...
 
         li
           a href="/laptop/laptop"Laptops
```
Comment (atainton) [inline app/views/shared/_nav.html.slim:29]: laptops_path

File: db/schema.rb [hunk 11]
```diff
@@ -11,7 +11,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160127065529) do
+ActiveRecord::Schema.define(version: 20160401125413) do
+
+  create_table "employees", force: :cascade do |t|
+    t.string   "fname"
```
Comment (atainton) [inline db/schema.rb]: please use better naming conventions

File: app/assets/stylesheets/form.scss (hunk 1 above)
Comment (danielsteenberg) [inline app/assets/stylesheets/form.scss]: deleted

File: app/assets/stylesheets/inputform.scss (hunk 2 above)
Comment (danielsteenberg) [inline app/assets/stylesheets/inputform.scss:1]: Done

File: spec/models/employee_spec.rb [hunk 12]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe Employee do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (atainton) [inline spec/models/employee_spec.rb:4]: are you going to write specs

File: app/assets/stylesheets/inputform.scss (hunk 3 above)
Comment (danielsteenberg) [inline app/assets/stylesheets/inputform.scss]: Made lower case

File: app/views/employees/edit.html.slim (hunk 4 above)
Comment (danielsteenberg) [inline app/views/employees/edit.html.slim:1]: 2 tab spaces and done

File: app/views/employees/edit.html.slim (hunk 5 above)
Comment (danielsteenberg) [inline app/views/employees/edit.html.slim]: want to use the button method to create buttons for users to click on, does this need to be changed

File: app/views/employees/index.html.slim (hunk 6 above)
Comment (danielsteenberg) [inline app/views/employees/index.html.slim]: How do i update this to change throughout the whole app?

File: app/views/employees/index.html.slim (hunk 7 above)
Comment (danielsteenberg) [inline app/views/employees/index.html.slim]: would like to use buttons

File: app/views/employees/index.html.slim (hunk 6 above)
Comment (danielsteenberg) [inline app/views/employees/index.html.slim]: sorted

File: app/views/employees/new.html.slim (hunk 8 above)
Comment (danielsteenberg) [inline app/views/employees/new.html.slim]: one equal set

File: app/views/shared/_nav.html.slim (hunk 10 above)
Comment (danielsteenberg) [inline app/views/shared/_nav.html.slim:29]: Doesnt recognise path no controller setup, will do that on next PR

File: app/views/shared/_nav.html.slim (hunk 9 above)
Comment (danielsteenberg) [inline app/views/shared/_nav.html.slim]: sorted

File: app/assets/stylesheets/inputform.scss (hunk 2 above)
Comment (atainton) [inline app/assets/stylesheets/inputform.scss:1]: bro check this file you can see its still off

File: app/views/employees/edit.html.slim (hunk 4 above)
Comment (atainton) [inline app/views/employees/edit.html.slim:1]: not two tab spaces, two space bar spaces

File: app/views/employees/edit.html.slim (hunk 5 above)
Comment (atainton) [inline app/views/employees/edit.html.slim]: It better to use link to with the employee path

Comment (atainton) [inline app/views/employees/edit.html.slim]: rails does that for you

File: app/views/employees/show.html.slim [hunk 13]
```diff
@@ -0,0 +1,16 @@
+
+.page-header
+    h1 Employee Details
+.container
+	p#notice = notice
+	
+	.form
+		p
+  		strong First Name:
+  		= @employee.first_name
```
Comment (atainton) [inline app/views/employees/show.html.slim:10]: much better

File: app/views/employees/show.html.slim [hunk 14]
```diff
@@ -0,0 +1,16 @@
...
+		p
+  		strong Last Name:
+  		= @employee.last_name
+
+		= button_to 'Edit', edit_employee_path(@employee), :method => :get
```
Comment (atainton) [inline app/views/employees/show.html.slim]: = link_to 'Edit', edit_employee_path(@employee)

File: spec/models/employee_spec.rb (hunk 12 above)
Comment (atainton) [inline spec/models/employee_spec.rb:4]: ??

File: app/views/employees/edit.html.slim [hunk 15]
```diff
@@ -4,7 +4,7 @@
 	.float-left
 		== render 'form'
 
-	= button_to 'Show', @employee, :method => :get
+	= link_to 'Show', @employee, class: 'btn btn-primary'
 	'|
-	= button_to 'Back', employees_path, :method => :get
+	= link_to 'Done', employees_path, class: 'btn btn-primary'
```
Comment (jamieandyou) [inline app/views/employees/edit.html.slim:9]: Nice work on the links and button classes :+1:

File: db/migrate/20160404125552_add_column.rb [hunk 16]
```diff
@@ -0,0 +1,5 @@
+class addColumn < ActiveRecord::Migration
+  def change
+  	add_column :employees, :department, :string
+  end
+end
```
Comment (jamieandyou) [inline db/migrate/20160404125552_add_column.rb]: why are there two of these migration files?

File: db/migrate/20160405072347_add_column_name.rb [hunk 17]
```diff
@@ -0,0 +1,5 @@
+class AddColumnName < ActiveRecord::Migration
+  def change
+  	add_column :employees, :department, :string
+  end
+end
```
Comment (jamieandyou) [inline db/migrate/20160405072347_add_column_name.rb:5]: why are there two of these migration files?

File: app/views/employees/index.html.slim [hunk 18]
```diff
@@ -6,23 +6,27 @@
...
             td = employee.first_name
             td = employee.last_name
+            td = employee.department
             td 
-              = button_to 'view', employee, :method => :get
-              = button_to 'Edit', edit_employee_path(employee), :method => :get
-              = button_to 'Delete', employee, data: {:confirm => 'Are you sure?'}, :method => :delete
+              = link_to 'view', employee_path(employee.id), class: 'btn btn-primary'
+              = link_to 'Edit', edit_employee_path(employee), class: 'btn btn-primary'
+              = link_to 'Delete', employee, data: { :confirm => 'Are you sure?' } , :method => :delete, class: 'btn btn-primary'
```
Comment (jamieandyou) [inline app/views/employees/index.html.slim]: Remove space after last curly brace
data: { :confirm => 'Are you sure?' },

File: app/views/employees/index.html.slim [hunk 19]
```diff
@@ -0,0 +1,38 @@
...
+        - @employees.each do |employee|
+          tr
+            td = employee.id
+            td = employee.first_name
+            td = employee.last_name
+            td = employee.department
+            td 
+              = link_to 'view', employee_path(employee.id), class: 'btn btn-primary'
+              = link_to 'Edit', edit_employee_path(employee), class: 'btn btn-primary'
+              = link_to 'Delete', employee, data: { :confirm => 'Are you sure?' } , :method => :delete, class: 'btn btn-primary'
```
Comment (jamieandyou) [inline app/views/employees/index.html.slim]: Remove space after last curly brace
data: { :confirm => 'Are you sure?' },

File: app/views/employees/show.html.slim [hunk 20]
```diff
@@ -0,0 +1,19 @@
...
+  		= @employee.first_name
+		p
+  		strong Last Name:
+  		= @employee.last_name
+  		p
+  		strong Department:
+  		= @employee.department
+
+	= link_to 'Edit', edit_employee_path(@employee), class: 'btn btn-primary'
+	= link_to 'Done', employees_path, class: 'btn btn-primary'
```
Comment (jamieandyou) [inline app/views/employees/show.html.slim:19]: Nice work on the links and button classes :+1:

File: db/migrate/20160405072347_add_column_name.rb (hunk 17 above)
Comment (danielsteenberg) [inline db/migrate/20160405072347_add_column_name.rb:5]: deleted one, had the same id as previous one so had to create another lol, deleted the duplicate

File: db/migrate/20160404125552_add_column.rb (hunk 16 above)
Comment (danielsteenberg) [inline db/migrate/20160404125552_add_column.rb]: deleted one, had the same id as previous one so had to create another lol, deleted the duplicate

File: app/views/employees/index.html.slim (hunk 18 above)
Comment (danielsteenberg) [inline app/views/employees/index.html.slim]: sorted

File: app/views/employees/index.html.slim (hunk 19 above)
Comment (danielsteenberg) [inline app/views/employees/index.html.slim]: sorted

File: app/views/employees/index.html.slim [hunk 21]
```diff
@@ -19,10 +19,10 @@
             td = employee.first_name
             td = employee.last_name
             td = employee.department
-            td 
+            td
               = link_to 'view', employee_path(employee.id), class: 'btn btn-primary'
               = link_to 'Edit', edit_employee_path(employee), class: 'btn btn-primary'
-              = link_to 'Delete', employee, data: { :confirm => 'Are you sure?' } , :method => :delete, class: 'btn btn-primary'
+              = link_to 'Delete', employee, data: {:confirm=>'Are you sure?'}, :method => :delete, class: 'btn btn-primary'
```
Comment (jamieandyou) [inline app/views/employees/index.html.slim]: It's nice to have those other spaces, it was just the space off the back that needed to come off.
{ :confirm => 'Are you sure?' },

File: app/views/employees/index.html.slim [hunk 22]
```diff
@@ -0,0 +1,38 @@
...
+        - @employees.each do |employee|
+          tr
+            td = employee.id
+            td = employee.first_name
+            td = employee.last_name
+            td = employee.department
+            td
+              = link_to 'view', employee_path(employee.id), class: 'btn btn-primary'
+              = link_to 'Edit', edit_employee_path(employee), class: 'btn btn-primary'
+              = link_to 'Delete', employee, data: {:confirm=>'Are you sure?'}, :method => :delete, class: 'btn btn-primary'
```
Comment (jamieandyou) [inline app/views/employees/index.html.slim]: It's nice to have those other spaces, it was just the space off the back that needed to come off.
{ :confirm => 'Are you sure?' },

Comment (danielsteenberg) [inline app/views/employees/index.html.slim]: correct the spacing

File: app/assets/stylesheets/inputform.scss [hunk 23]
```diff
@@ -0,0 +1,22 @@
+.form {
+	border: solid #000000;
+	width: 50%;
+	padding: 40px;
+}
+
+.button_to {
+	text-align: left;
```
Comment (jamieandyou) [inline app/assets/stylesheets/inputform.scss:8]: fix indentation

**Comment by jamieandyou** [comment]
:+1:

**Pull request merged**